packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -2006,7 +2006,6 @@ describe('TransactionController', () => {
         gasPrice: '0x1',
         to: ACCOUNT_MOCK,
         value: '0x0',
-        type: TransactionType.simpleSend,
```

Comment:
Good spot 😄  It's a fun one having a `type` property in both the parameters and the options.